tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -175,10 +146,12 @@ describe('IssueList', function () {
 
     /* helpers */
     const getSavedSearchTitle = w =>
-      w.find('SavedSearchTab DropdownMenu a').text().trim();
+      w.queryByTestId('saved-search-title')?.textContent.trim();
 
     const getSearchBarValue = w =>
-      w.find('SmartSearchBarContainer textarea').prop('value').trim();
+      w
+        .queryByPlaceholderText('Search for events, users, tags, and more')
```

Comment:
should we use `findByPlaceholderText` instead of `queryByPlaceholderText` ?

Reply:
We were using this like

```javascript
await waitFor(() => expect(getSearchBarValue(wrapper)).toBe('is:unresolved'));
```

I've updated it to be this instead

```javascript
await wrapper.findByDisplayValue('is:unresolved');
```